tests/adapters/test_232ct_with_device.py

```diff
@@ -79,4 +81,4 @@ def test_current_reading(self, make_clean_instrument):
         dmm.resolution = 3
         dmm.mode = "DCI"
         value = dmm.measure_DCI
-        assert type(value) is float
+        assert type(value) is float
```

Comment:
it is better to use `isinstance(value, float)` than doing type checks.

Reply:
Thanks for the recommenation, adopted